src/test/common/services/tokenApi.test.js

```diff
@@ -129,14 +114,31 @@ describe('UserSessions', () => {
       MFAToken: 87654321,
       NumAttempts: 5,
     });
+
     dbStub.sequelize.models.UserSessions.findOne = findOneStubFail;
-    tokenApi.validateMfaToken('myemail@email.com', 87654321)
-      .then(() => {
-        expect(false).to.be.true;
-      })
-      .catch(() => {
-        expect(true).to.be.true;
-        done();
-      });
+    try {
+      await tokenApi.validateMfaToken('myemail@email.com', 87654321);
+    } catch (err) {
+      expect(err.message).to.equal(`MFA token verification attempts exceeded, maximum limit ${MFA_TOKEN_MAX_ATTEMPTS}`);
+    }
+  });
+
+  it('Should reject a correctly entered token that has been expired', async () => {
+    const findOneStubFail = sinon.stub().resolves({
```

Comment:
If I had to be pedantic, its not a "findOneStubFail", its a "findOneStubExpired"?